### DISCUSSION TITLE
周年庆的活动挑战结算没有点击

### DISCUSSION THREAD

**Discussion by iygind** [lisai9093/YYS#10] [Q&A] [answered]
我刚刚使用发现周年庆活动对局结算需要 点击两次，分别是获得御魂等奖励和ying，领取御魂这个就没有点击

**Comment by lisai9093** [comment]
我还没有发现这个问题，会影响使用么？

**Reply by iygind** [reply]
![Snipaste_2023-10-03_01-00-35](https://github.com/lisai9093/YYS/assets/85872101/a5e30e53-5bdb-44fa-914d-0e3dbf77d277)

**Reply by iygind** [reply]
就是这个获得奖励是没有反应的 我是否需要改文件里的这个图片，但我不知道是那个文件，谢谢。之前没有上git，即使现在要结束了，我还是想知道并举一反三

**Reply by lisai9093** [reply, marked as answer]
你可以试试替换"jixu" 或者加入新图片。“jixu”对应的是最下方的“点击屏幕继续”，这样即使上面被挡住也没问题。

**Reply by iygind** [reply]
好的 谢谢 我用的是模拟器，新区现在用不了桌面了

**Comment by lisai9093** [comment]
对了，我看你用的模拟器，我测试用的是桌面版，所以有些画面可能会不一样
